roles/common.nix

```diff
@@ -7,11 +7,6 @@
     ./users.nix
   ];
 
-  # Nicer interactive shell
-  programs.fish.enable = true;
-  # And for the zsh peeps
-  programs.zsh.enable = true;
-
```

Comment:
Do we have this still for build01?

Reply:
Do we need other shells? e.g. the aarch64 build box doesn't have them.

Fish seems to be building a bunch of completion stuff as separate drvs which seems a bit pointless on these machines.

Reply:
> Fish seems to be building a bunch of completion stuff as separate drvs which seems a bit pointless.

It's because it can generate completions from man pages.

Reply:
Moved fish/zsh to roles/buillder for build01 and disabled the generated completion.